python/src/lib.rs

```diff
@@ -1232,6 +1232,19 @@ impl RawDeltaTable {
         self._table.state = table.state;
         Ok(serde_json::to_string(&metrics).unwrap())
     }
+
+    pub fn transaction_versions(&self) -> HashMap<String, String> {
+        self._table
+            .get_app_transaction_version()
+            .iter()
+            .map(|(app_id, transaction)| {
+                (
+                    app_id.to_owned(),
+                    serde_json::to_string(transaction).unwrap(),
```

Comment:
Maybe return the PyTransaction here

Reply:
@ion-elgreco updated! I made `PyTransaction` a `pyclass` so we can pass it to/from python more easily